EnglishTranslation-master/SKILL.tsv

```diff
@@ -4531,9 +4531,9 @@ SKILL_20150714_004530	Increases the damage dealt on an enemy with [Stone Shot] b
 SKILL_20150714_004531	Increases the damage dealt on an enemy with [Rapid Fire] by 1% per attribute level.
 SKILL_20150714_004532	Increases the damage dealt on an enemy with [Teardown] by 1% per attribute level.
 SKILL_20150714_004533	Increases the duration of [Scatter Caltrops] by 1 second per attribute level.
-SKILL_20150714_004534	There is a 8% chance per attribute level to  be afflicted with [Bleeding] when stepping on a caltrop.
+SKILL_20150714_004534	There is a 8% chance per attribute level that caltrops will cause [Bleeding] when stepped on.
```

Comment:
`8% chance per attribute level that caltrops will cause [Bleeding] when stepped on.`